cmd/buildah/addcopy.go

```diff
@@ -38,20 +34,13 @@ func addAndCopyCmd(c *cli.Context, extractLocalArchives bool) error {
 		name = c.String("name")
 	}
 	root := c.String("root")
-	link := ""
-	if c.IsSet("link") {
-		link = c.String("link")
-		if link == "" {
-			return fmt.Errorf("link location can not be empty")
-		}
-	}
 	dest := ""
 	if c.IsSet("dest") {
 		dest = c.String("dest")
 	}
-	if name == "" && root == "" && link == "" {
+	if name == "" && root == "" {
 		if len(args) == 0 {
-			return fmt.Errorf("either a container name or --root or --link, or some combination, must be specified")
+			return fmt.Errorf("either a container name or --root, or some combination, must be specified")
```

Comment:
Thanks for the explanation.  I still think we should call it a container id in the documentation/calls/params and then in the help explain that a name will work too.  I'll head nod to whatever you and Dan this is best on this, but that's my $.02.